content/en/docs/migration/_index.md

```diff
@@ -3,3 +3,29 @@ title: Migration
 description: How to migrate to OpenTelemetry
 weight: 50
 ---
+
+## OpenTracing and OpenCensus
+
+OpenTelemetry was created as a merger of OpenTracing and OpenCensus. From the
+start OpenTelemetry was considered [to be the next major version of both
+OpenTracing and OpenCensus][]. Because of that, one of the [key goals][] of the
+OpenTelemetry project is to provide backward compatibility with OpenCensus and a
```

Comment:
Why only mention OpenCensus here in terms of backwards compatibility?

Reply:
good call! fixed